packages/tasit-action/package.json

```diff
@@ -38,6 +38,7 @@
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "mocha": "^6.0.2",
+    "node-fetch": "^2.3.0",
```

Comment:
Since I've added that as a `devDependency` only I wasn't worried about the better option for `react-native` app, I've just picked the one that seems most popular.
I just took a quick look at `fribee` and `axios` and I think that:
`frisbee` is a good choice for simple rest API consumption;
`axios` seems more complete and better for more complex needs.
If will be better to us pick one module that will be used on our apps too, I think that `axios` can be a better option since our needs aren't totally clear at the moment.

Reply:
Thanks for the detailed thoughts. That all makes sense. If you like we can extract this to an issue and keep it as-is for now.

Reply:
Extracted to: https://github.com/tasitlabs/TasitSDK/issues/294

Reply:
Thanks. Resolving this.